### DISCUSSION TITLE
Style specific pages

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#8717] [Ideas]
I'd like to hook into

```php
FilamentView::registerRenderHook('panels::styles.after', ...)
```
but I cannot pass any scopes since the hooks inside 
```
vendor/filament/filament/resources/views/components/layout/base.blade.php
```
are not passed any `renderHookScopes` (since it happens inside the base layout).

Any way to do this nicely? I want to style some pages without affecting any others.

**Comment by ousid** [comment]
Take a look [here](https://filamentphp.com/docs/3.x/support/style-customization#overview)

**Comment by ghost** [comment]
I need css classes on higher levels (like on `<body>`) that are specific to the page. Using general styles like on a `.fi-ctn` target does not work since those classes are often times shared across pages.

**Reply by cawecoy** [reply]
Have you got to find a solution?
